core/src/main/resources/data/upgrade_5.3.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE software ALTER COLUMN versionEndExcluding SET DATA TYPE VARCHAR(75);
+ALTER TABLE software ALTER COLUMN versionEndIncluding SET DATA TYPE VARCHAR(75);
+ALTER TABLE software ALTER COLUMN versionStartExcluding SET DATA TYPE VARCHAR(75);
+ALTER TABLE software ALTER COLUMN versionStartIncluding SET DATA TYPE VARCHAR(75);
```

Comment:
```suggestion
ALTER TABLE software ALTER COLUMN versionEndExcluding SET DATA TYPE CHARACTER LARGE OBJECT;
ALTER TABLE software ALTER COLUMN versionEndIncluding SET DATA TYPE VARCHAR(75);
ALTER TABLE software ALTER COLUMN versionStartExcluding SET DATA TYPE CHARACTER LARGE OBJECT;
ALTER TABLE software ALTER COLUMN versionStartIncluding SET DATA TYPE CHARACTER LARGE OBJECT);
```

and lets forgot about that kind of issue forever.

Reply:
I am already working on this. There are a few other changes as well.